java/client/src/main/java/glide/api/commands/SetBaseCommands.java

```diff
@@ -95,4 +95,20 @@ public interface SetBaseCommands {
      * }</pre>
      */
     CompletableFuture<Boolean> sismember(String key, String member);
+
+    /**
+     * Stores the members of the difference between the first set and all the successive sets in
+     * <code>keys</code> into a new set at <code>destination</code>.
+     *
+     * @see <a href="https://redis.io/commands/sdiffstore/">redis.io</a> for details.
+     * @param destination The key of the destination set.
+     * @param keys The keys from which to retrieve the set members.
+     * @return The number of elements in the resulting set.
```

Comment:
Do you want to talk about if a set does not exist it is treated as an empty set?

Reply:
https://github.com/aws/glide-for-redis/pull/1243/files#diff-20dfe9bdf682dbfb9c8b2fdda8e5c534d479623e35a5182761d231d747fbdc82R461 do you want to keep documentation similar to this to be consistent?

Reply:
Is it better now?

Reply:
Do you want to talk about if a set does not exist it is treated as an empty set? Do you want to add a detail about this other than looks good.

Reply:
It is not said in the official docs unfortunately

Reply:
https://redis.io/docs/latest/commands/zdiffstore/ it is mentioned here. Your call if you want to add it.